app/models/resource.py

```diff
@@ -0,0 +1,30 @@
+from .. import db
+
+
+class Resource(db.Model):
+    __tablename__ = 'resources'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(64))
+    address = db.relationship('Address', backref='resource',
```

Comment:
Ben brings up a good point here. I would say we probably want this to be a `joined